vassal-app/src/main/java/VASSAL/tools/image/svg/SVGImageUtils.java

```diff
@@ -78,60 +79,55 @@ public static Dimension getImageSize(InputStream in) throws IOException {
    * @return the image dimensions
    * @throws IOException if the image cannot be read
    */
-  public static Dimension getImageSize(String name, InputStream in)
-                                                          throws IOException {
+  public static Dimension getImageSize(String name, InputStream in) throws IOException {
     // get the SVG
     final Document doc;
     try (in) {
       synchronized (factory) {
         doc = factory.createDocument(null, in);
       }
-    }
+    } 
     catch (FileNotFoundException e) {
       throw new ImageNotFoundException(name, e);
-    }
+    } 
     catch (DOMException | IOException e) {
       throw new ImageIOException(name, e);
     }
 
     // get the default image width and height
     final Element root = doc.getDocumentElement();
     try {
-      final int width = (int) (Float.parseFloat(
-        root.getAttributeNS(null, "width").replaceFirst("px", ""))+0.5);
-      final int height = (int) (Float.parseFloat(
-        root.getAttributeNS(null, "height").replaceFirst("px", ""))+0.5);
+      final int width = (int) (Float.parseFloat(root.getAttributeNS(null, "width").replaceFirst("px", "")) + 0.5);
+      final int height = (int) (Float.parseFloat(root.getAttributeNS(null, "height").replaceFirst("px", "")) + 0.5);
 
       return new Dimension(width, height);
-    }
+    } 
     catch (NumberFormatException e) {
-      throw new ImageIOException(name, e);
+      throw new ImageIOException(name + " (Only 'px' units supported for height and width in SVG files)", e);
```

Comment:
Much better overall, but what is that extra trailing whitespace on line 104? There seems to be a few of them added to this file.